components/user_settings/display/manage_timezones/manage_timezones.tsx

```diff
@@ -21,9 +23,9 @@ type Props = {
     useAutomaticTimezone: boolean;
     automaticTimezone: string;
     manualTimezone: string;
-    timezones: string[];
+    timezones: Timezone[];
+    timezoneLabel: string;
```

Comment:
Do we have to pass the label in here or could we just figure it out from the initial timezone object passed in?

Reply:
I got the timezoneLabel using a memoized selector, it's just a little bit more efficient rather than recalculating the label in the components each time